src/relative-paths.test.ts

```diff
@@ -0,0 +1,50 @@
+/* global test, expect, afterEach */
+// https://stackoverflow.com/questions/39415661
```

Comment:
You do not need this reference for the import-require statement.
Its part of the official typescript documentation. No need to quote these.

https://www.typescriptlang.org/docs/handbook/modules.html